### PR TITLE
doc: transactional load data stmt

### PR DESCRIPTION
**Pull request by ekexium**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tidb/issues/49078

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by lance6716 — APPROVED** [review]

File: sql-statements/sql-statement-load-data.md [hunk 1]
```diff
@@ -120,8 +120,12 @@ TiDB 中的 `LOAD DATA` 语句语法上兼容 MySQL（除字符集选项被解
...
 > - 从 TiDB v4.0.0 开始一直到 TiDB v6.6.0 的版本，TiDB 默认在一个事务中提交所有行。
 > - 从 TiDB v4.0.0 及以前版本升级后，可能出现 `ERROR 8004 (HY000) at line 1: Transaction is too large, size: 100000058` 错误。要解决该问题，建议调大 `tidb.toml` 文件中的 [`txn-total-size-limit`](/tidb-configuration-file.md#txn-total-size-limit) 值。如果无法增加此限制，还可以将 [`tidb_dml_batch_size`](/system-variables.md#tidb_dml_batch_size) 的值设置为 `20000` 来恢复升级前的行为。注意自 v7.0.0 起，`tidb_dml_batch_size` 对 `LOAD DATA` 语句不再生效。
-> - 无论以多少行为一个事务提交，`LOAD DATA` 都不会被显式事务中的 [`ROLLBACK`](/sql-statements/sql-statement-rollback.md) 语句回滚。
-> - `LOAD DATA` 语句始终以乐观事务模式执行，不受 TiDB 事务模式设置的影响。
+> - 在小于 v7.6.0 的版本中，无论以多少行为一个事务提交，`LOAD DATA` 都不会被显式事务中的 [`ROLLBACK`](/sql-statements/sql-statement-rollback.md) 语句回滚。
+> - 在小于 v7.6.0 的版本中，`LOAD DATA` 语句始终以乐观事务模式执行，不受 TiDB 事务模式设置的影响。
+> - 自 TiDB v7.6.0 起，`LOAD DATA` 在事务内的地位与其它普通 DML 一致。
+>   - `LOAD DATA` 语句本身不会提交当前事务，不会开启新事务。
+>   - `LOAD DATA` 语句会受到事务模式设置（乐观/悲观）影响。
+>   - 事务内的 `LOAD DATA` 语句可以被事务的 [`ROLLBACK`](/sql-statements/sql-statement-rollback.md) 语句回滚。
```
Comment (Oreoxmt) [inline sql-statements/sql-statement-load-data.md:128]:
```suggestion
> - 在 TiDB v7.6.0 之前的版本中，无论以多少行为一个事务提交，`LOAD DATA` 都不会被显式事务中的 [`ROLLBACK`](/sql-statements/sql-statement-rollback.md) 语句回滚。
> - 在 TiDB v7.6.0 之前的版本中，`LOAD DATA` 语句始终以乐观事务模式执行，不受 TiDB 事务模式设置的影响。
> - 从 TiDB v7.6.0 开始，`LOAD DATA` 在事务中的处理与其它普通 DML 一致：
>   - `LOAD DATA` 语句本身不会提交当前事务，也不会开启新事务。
>   - `LOAD DATA` 语句会受 TiDB 事务模式设置（乐观/悲观）影响。
>   - 事务内的 `LOAD DATA` 语句可以被显式事务中的 [`ROLLBACK`](/sql-statements/sql-statement-rollback.md) 语句回滚。
```

Comment (ekexium) [reply sql-statements/sql-statement-load-data.md:128]: “显式事务” 包括non-autocommit，但不通过 begin 语句开始的事务吗？我的理解是不包括。如果是这个含意的话显式事务不够精确。

Comment (Oreoxmt) [reply sql-statements/sql-statement-load-data.md:128]: 已更新为“可以被事务的 [`ROLLBACK`](/sql-statements/sql-statement-rollback.md) 语句回滚。” PTAL again

**Review by Oreoxmt — APPROVED** [review]

File: sql-statements/sql-statement-load-data.md [hunk 2]
```diff
@@ -120,8 +120,12 @@ TiDB 中的 `LOAD DATA` 语句语法上兼容 MySQL（除字符集选项被解
...
 > - 从 TiDB v4.0.0 开始一直到 TiDB v6.6.0 的版本，TiDB 默认在一个事务中提交所有行。
 > - 从 TiDB v4.0.0 及以前版本升级后，可能出现 `ERROR 8004 (HY000) at line 1: Transaction is too large, size: 100000058` 错误。要解决该问题，建议调大 `tidb.toml` 文件中的 [`txn-total-size-limit`](/tidb-configuration-file.md#txn-total-size-limit) 值。如果无法增加此限制，还可以将 [`tidb_dml_batch_size`](/system-variables.md#tidb_dml_batch_size) 的值设置为 `20000` 来恢复升级前的行为。注意自 v7.0.0 起，`tidb_dml_batch_size` 对 `LOAD DATA` 语句不再生效。
-> - 无论以多少行为一个事务提交，`LOAD DATA` 都不会被显式事务中的 [`ROLLBACK`](/sql-statements/sql-statement-rollback.md) 语句回滚。
-> - `LOAD DATA` 语句始终以乐观事务模式执行，不受 TiDB 事务模式设置的影响。
+> - 在 TiDB v7.6.0 之前的版本中，无论以多少行为一个事务提交，`LOAD DATA` 都不会被显式事务中的 [`ROLLBACK`](/sql-statements/sql-statement-rollback.md) 语句回滚。
+> - 在 TiDB v7.6.0 之前的版本中，`LOAD DATA` 语句始终以乐观事务模式执行，不受 TiDB 事务模式设置的影响。
+> - 从 TiDB v7.6.0 开始，`LOAD DATA` 在事务中的处理与其它普通 DML 一致：
+>   - `LOAD DATA` 语句本身不会提交当前事务，也不会开启新事务。
+>   - `LOAD DATA` 语句会受 TiDB 事务模式设置（乐观/悲观）影响。
+>   - 事务内的 `LOAD DATA` 语句可以被事务的 [`ROLLBACK`](/sql-statements/sql-statement-rollback.md) 语句回滚。
```
Comment (Oreoxmt) [inline sql-statements/sql-statement-load-data.md:128]:
```suggestion
>     - `LOAD DATA` 语句本身不会提交当前事务，也不会开启新事务。
>     - `LOAD DATA` 语句会受 TiDB 事务模式设置（乐观/悲观）影响。
>     - 事务内的 `LOAD DATA` 语句可以被事务的 [`ROLLBACK`](/sql-statements/sql-statement-rollback.md) 语句回滚。
```

File: sql-statements/sql-statement-load-data.md [hunk 3]
```diff
@@ -120,8 +120,12 @@ TiDB 中的 `LOAD DATA` 语句语法上兼容 MySQL（除字符集选项被解
 > - 在 TiDB v4.0.0 之前的版本中，`LOAD DATA` 语句每 20000 行进行一次提交。
 > - 从 TiDB v4.0.0 开始一直到 TiDB v6.6.0 的版本，TiDB 默认在一个事务中提交所有行。
 > - 从 TiDB v4.0.0 及以前版本升级后，可能出现 `ERROR 8004 (HY000) at line 1: Transaction is too large, size: 100000058` 错误。要解决该问题，建议调大 `tidb.toml` 文件中的 [`txn-total-size-limit`](/tidb-configuration-file.md#txn-total-size-limit) 值。如果无法增加此限制，还可以将 [`tidb_dml_batch_size`](/system-variables.md#tidb_dml_batch_size) 的值设置为 `20000` 来恢复升级前的行为。注意自 v7.0.0 起，`tidb_dml_batch_size` 对 `LOAD DATA` 语句不再生效。
-> - 无论以多少行为一个事务提交，`LOAD DATA` 都不会被显式事务中的 [`ROLLBACK`](/sql-statements/sql-statement-rollback.md) 语句回滚。
-> - `LOAD DATA` 语句始终以乐观事务模式执行，不受 TiDB 事务模式设置的影响。
+> - 在 TiDB v7.6.0 之前的版本中，无论以多少行为一个事务提交，`LOAD DATA` 都不会被显式事务中的 [`ROLLBACK`](/sql-statements/sql-statement-rollback.md) 语句回滚。
+> - 在 TiDB v7.6.0 之前的版本中，`LOAD DATA` 语句始终以乐观事务模式执行，不受 TiDB 事务模式设置的影响。
+> - 从 TiDB v7.6.0 开始，`LOAD DATA` 在事务中的处理与其它普通 DML 一致：
```
Comment (qiancai) [inline sql-statements/sql-statement-load-data.md:125]:
```suggestion
> - 从 TiDB v7.6.0 开始，`LOAD DATA` 在事务中与其它普通 DML 的处理方式一致：
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Pull request merged**